symphony-bdk-core/src/test/java/com/symphony/bdk/core/service/stream/StreamServiceTest.java

```diff
@@ -0,0 +1,309 @@
+import com.symphony.bdk.gen.api.model.V3RoomSearchResults;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class StreamServiceTest {
+
+  private static final String V1_IM_CREATE = "/pod/v1/im/create";
+  private static final String V1_IM_CREATE_ADMIN = "/pod/v1/admin/im/create";
+  private static final String V1_ROOM_SET_ACTIVE = "/pod/v1/room/{id}/setActive";
+  private static final String V1_ROOM_SET_ACTIVE_ADMIN = "/pod/v1/admin/room/{id}/setActive";
+  private static final String V1_STREAM_LIST = "/pod/v1/streams/list";
+  private static final String V1_STREAM_ATTACHMENTS = "/pod/v1/streams/{sid}/attachments";
+  private static final String V1_STREAM_MEMBERS = "/pod/v1/admin/stream/{id}/membership/list";
+  private static final String V2_STREAM_INFO = "/pod/v2/streams/{sid}/info";
+  private static final String V2_STREAM_LIST_ADMIN = "/pod/v2/admin/streams/list";
+  private static final String V3_ROOM_CREATE = "/pod/v3/room/create";
+  private static final String V3_ROOM_SEARCH = "/pod/v3/room/search";
+  private static final String V3_ROOM_INFO = "/pod/v3/room/{id}/info";
+  private static final String V3_ROOM_UPDATE = "/pod/v3/room/{id}/update";
+
+  private StreamService service;
+  private MockApiClient mockApiClient;
+
+  @BeforeEach
+  void setUp() {
+    this.mockApiClient = new MockApiClient();
+    AuthSession authSession = mock(AuthSession.class);
+    ApiClient podClient = mockApiClient.getApiClient("/pod");
+    this.service = new StreamService(new StreamsApi(podClient), authSession);
+
+    when(authSession.getSessionToken()).thenReturn("1234");
+    when(authSession.getKeyManagerToken()).thenReturn("1234");
+  }
+
+  @Test
+  void createIMorMIMTest() {
+    this.mockApiClient.onPost(V1_IM_CREATE, "{\"id\": \"xhGxbTcvTDK6EIMMrwdOrX___quztr2HdA\"}");
+
+    Stream stream = this.service.create(Arrays.asList(7215545078541L, 7215512356741L));
+
+    assertEquals(stream.getId(), "xhGxbTcvTDK6EIMMrwdOrX___quztr2HdA");
+  }
+
+  @Test
+  void createIMorMIMTestFailed() {
+    this.mockApiClient.onPostFailed(400, V1_IM_CREATE, "{}");
+
+    assertThrows(ApiRuntimeException.class, () -> this.service.create(Arrays.asList(7215545078541L, 7215512356741L)));
+  }
+
+  @Test
+  void createIMTest() {
+    this.mockApiClient.onPost(V1_IM_CREATE, "{\"id\": \"xhGxbTcvTDK6EIMMrwdOrX___quztr2HdA\"}");
+
+    Stream stream = this.service.create(7215545078541L);
+
+    assertEquals(stream.getId(), "xhGxbTcvTDK6EIMMrwdOrX___quztr2HdA");
+  }
+
+  @Test
+  void createIMTestFailed() {
+    this.mockApiClient.onPostFailed(400, V1_IM_CREATE, "{}");
+
+    assertThrows(ApiRuntimeException.class, () -> this.service.create(7215545078541L));
+  }
+
+  @Test
+  void createRoomChatTest() throws IOException {
+    this.mockApiClient.onPost(V3_ROOM_CREATE, ResResponseHelper.readResResponseFromClasspath(
```

Comment:
Could you rename `ResResponseHelper.readResResponseFromClasspath` to `JsonHelper.readFromClasspath`?